api/README.md

```diff
@@ -144,3 +143,36 @@ The list of available handlebars variables:
   - `name`: The service name which is equivalent to the network alias
   - `port`: The exposed port of the service
   - `type`: The type of service. For example, `instance`, `replica`, `app-companion`, or `service-companion`.
+
+## Hooks
```

Comment:
I first thought a hook is for a complete deployment but not just for one service per deployment. Maybe this could be made a little clearer here.

Reply:
Yes, it is for a complete deployment. The JS code receives the `appName` and an array of all service configurations. I'll check if I can improve the documentation. Otherwise, do you have a proposal how to clarify it?